src/elements-experimental/seat-reservation/seat-reservation/seat-reservation-base-element.ts

```diff
@@ -262,43 +264,49 @@ export class SeatReservationBaseElement extends LitElement {
       // Set maximum calculated coach width
       this.maxCalcCoachsWidth = currCalcTriggerPos;
 
-      // At the end of a scroll Events to a coach, the reached wagon is marked as selected
-      this.coachScrollArea.addEventListener('scrollend', () => {
-        const findScrollCoachIndex = this.isAutoScrolling
-          ? this.currSelectedCoachIndex
-          : this._getCoachIndexByScrollTriggerPosition();
+      this.coachScrollArea.addEventListener('scroll', () => {
+        // Timeout event handling to check if the scrolling has been completed.  -> scrollend
+        if (this._scrollTimeoutId) clearTimeout(this._scrollTimeoutId);
+        //If no further scoll event is fired, the last timeout is fired
+        this._scrollTimeoutId = window.setTimeout(() => this._handleCoachAreaScrollendEvent(), 150);
```

Comment:
if the purpose is to throttle the scroll event, we also have a throttle() function which you (maybe) could use.